pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -395,7 +395,7 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                 )
 
                 # We could refresh here too, but adding 3 saves a inventory request
-                candy = inventory.candies(True).get(pokemon.pokemon_id)
+                candy = inventory.candies().get(pokemon.pokemon_id).add(3)
```

Comment:
Pretty sure you can get the exact number of awarded candies in the return API call (currently 3 but could change in the future)